documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -3,21 +3,41 @@
 // assembly-security.adoc
 
 [id='proc-installing-certs-per-listener-{context}']
-= Providing your own Kafka listener certificates
+= Providing your own Kafka listener certificates for TLS authentication
 
-This procedure shows how to configure a listener to use your own private key and server certificate, called a xref:kafka-listener-certificates-{context}[Kafka listener certificate].
+[role="_abstract"]
+Listeners provide client access to Kafka brokers.
+You configure listeners in the `Kafka` resource, including the configuration required for client access using TLS authentication.
 
-Your client applications should use the CA public key as a trusted certificate in order to verify the identity of the Kafka broker.
+By default, the listeners use certificates signed by the internal self-signed CAs generated by Strimzi.
+A public CA certificate is generated by the Cluster Operator when it creates a Kafka cluster.
```

Comment:
Public CA would normally be understood as a well recognized CA such as LEt's encrypt, which is not the case here.

```suggestion
A CA certificate is generated by the Cluster Operator when it creates a Kafka cluster.
```